D2547.md

```diff
@@ -1301,7 +1292,7 @@ For example, consider:
 
 ```cpp
 template<typename T, typename U>
-virtual void swap(T&& t, U&& u) = 0;
+void swap(T&& t, U&& u) customisable;
```

Comment:
This is a case where I think the previous `= 0` syntax helped to indicate that there isn't a default implementation.

If I were just reading this as a programmer unfamiliar with the syntax, I might assume that this is a customisable function that has a default which is defined elsewhere.

Reply:
We'll just have to wait and see what the feedback is about this...